microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color.md

```diff
@@ -60,10 +60,224 @@ Each icon in the `icons` array should come with at least the `src` and `sizes` p
 
+    * 44x44 - app icon
+    * 55x55 - app icon 1.25x display scale
+    * 66x66 - app icon 1.5x display scale
+    * 88x88 - app icon 2x display scale
+    * 176x176 - app icon 4x display scale
+    * 71x71 - small tile
+    * 89x89 - small tile 1.25x display scale
+    * 107x107 - small tile 1.5x display scale
+    * 142x142 - small tile 2x display scale
+    * 284x284 - small tile 4x display scale
+    * 150x150 - medium tile
+    * 188x188 - medium tile 1.25x display scale
+    * 225x225 - medium tile 1.5x display scale
+    * 300x300 - medium tile 2x display scale
+    * 600x600 - medium tile 4x display scale
+    * 350x150 - wide tile
+    * 388x188 - wide tile 1.25x display scale
+    * 465x225 - wide tile 1.5x display scale
+    * 620x300 - wide tile 2x display scale
+    * 1240x600 - wide tile 4x display scale
+    * 310x310 - large tile
+    * 388x388 - large tile 1.25x display scale
+    * 465x465 - large tile 1.5x display scale
+    * 620x620 - large tile 2x display scale
+    * 1240x1240 - large tile 4x display scale
+    * 50x50 - store tile
+    * 63x63 - store tile 1.25x display scale
+    * 75x75 - store tile 1.5x display scale
+    * 100x100 - store tile 2x display scale
+    * 200x200 - store tile 4x display scale
+    * 620x300 - splash screen
+    * 775x375 - splash screen 1.25x display scale
+    * 930x450 - splash screen 1.5x display scale
+    * 1240x600 - splash screen 2x display scale
+    * 2480x1200 - splash screen 4x display scale
+    * 16x16 - target size for taskbar, start menu, task manager
+    * 20x20 - target size for taskbar, start menu, task manager
+    * 24x24 - target size for taskbar, start menu, task manager
+    * 30x30 - target size for taskbar, start menu, task manager
+    * 32x32 - target size for taskbar, start menu, task manager
+    * 36x36 - target size for taskbar, start menu, task manager
+    * 40x40 - target size for taskbar, start menu, task manager
+    * 44x44 - target size for taskbar, start menu, task manager
+    * 48x48 - target size for taskbar, start menu, task manager
+    * 60x60 - target size for taskbar, start menu, task manager
+    * 64x64 - target size for taskbar, start menu, task manager
+    * 72x72 - target size for taskbar, start menu, task manager
+    * 80x80 - target size for taskbar, start menu, task manager
+    * 96x96 - target size for taskbar, start menu, task manager
+    * 256x256 - target size for taskbar, start menu, task manager
+In level 4, you supply images for tiles with display scales and target size images for display in various surfaces in Windows, including taskbar, start menu, task manager, ALT+Tab task switcher, and more.
```

Comment:
This line can be removed now, since you added it at the top of the level 4 bullet list item.

Reply:
@captainbrosset This line?  In level 4, you supply images for tiles with display scales and target size images for display in various surfaces in Windows, including taskbar, start menu, task manager, ALT+Tab task switcher, and more.

Reply:
Yes, please delete this line, since it is already present earlier in the article, where the level 4 images are described (line 116).

Reply:
done